prometheus/metrics_exporter.go

```diff
@@ -8,15 +8,15 @@ import (
   "fmt"
   "time"
   "strconv"
-  "github.com/jesseadams/kafka-connect-monitoring-tools/common"
+  "kafka-connect-monitoring-tools/common"
```

Comment:
Please revert this back

Reply:
This needs to be built locally. I changed this because I had made changes to common.
I dont understand why you would want to go back out to github once you have the repo locally?

Reply:
This just effects the `go get` portion of a build. You don't have to do this every time. If you check this out to the same location in your `GOPATH` that go get would, you can easily do development on it with the original import string.